pkg/controller/controller_binding.go

```diff
@@ -257,6 +257,22 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 			BindResource: &osb.BindResource{AppGUID: &appGUID},
 		}
 
+		originatingIdentity, err := buildOriginatingIdentity(binding.Spec.UserInfo)
+		if err != nil {
+			s := fmt.Sprintf("Failed to prepare Binding originating identity\n%v\n %v", binding.Spec.UserInfo, err)
```

Comment:
It might be more readable if you put something in this message identifying the error as an error